web/src/components/Timer/Timer.tsx

```diff
@@ -32,15 +32,21 @@ export const Timer = ({
   return typeof window !== "undefined"
     ? createPortal(
         <Container>
-          <TimerContainer className={`duration-[${initialSeconds}s]`}>
+          {/* Timer container */}
+          <div 
+            style={{
+              animation: `move-animation ${initialSeconds}s linear infinite`,
```

Comment:
inline totally cool, but a neat trick you can do is to define a css variable and read from that. so in this case:
```
<div
  style={{ "--initialSeconds": initialSeconds }}
  className="animate-[move-animation_[var(--initialSeconds)]s_linear_infinite]"
>
```

Reply:
What an interesting workaround! I'll change it to use a css variable

Reply:
I managed to make it work like this 
` 
          <div
            style={{ "--initialSeconds": `${initialSeconds}s` } as React.CSSProperties}
            className="animate-[move-animation_var(--initialSeconds)_linear_infinite] [animation-iteration-count:1]"
          >
`